<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiClass</class>
 <widget class="QWidget" name="GuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gui</string>
  </property>
  <widget class="QComboBox" name="trackerType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mosse</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kcf</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tld</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MedianFlow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Csrt</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="whereTrack">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Original</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Transformovaný</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kruhy</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="whereTrackLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sledovaný obraz:</string>
   </property>
  </widget>
  <widget class="QLabel" name="trackerTypeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sledovač:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="firstX">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>6000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="firstY">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>6000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="secondX">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>6000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="secondY">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>6000</number>
   </property>
  </widget>
  <widget class="QLabel" name="cropLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ořez:</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstPointLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1) Bod</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstXlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x:</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstYlabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondXlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x:</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondYlable">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondPointLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2) Bod</string>
   </property>
  </widget>
  <widget class="QLabel" name="framesToWriteLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Počet snímku, které budou zapsány do souboru:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="framesToWrite">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="erodeSizeLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Velikost prvku eroze:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="erodeSize">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dilateSizeLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Velikost prvku dilatace:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="dilateSize">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="treshLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Práh:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="thresh">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="minAreaLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimální velikost objektu:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minArea">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Zpět</string>
   </property>
  </widget>
  <widget class="QLabel" name="previewImg">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="getPrew">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Náhled</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Gui.qrc"/>
 </resources>
 <connections/>
</ui>
